### PR TITLE
Add support for flake8 3.8.0

### LINKED ISSUE #35: flake8-pyi is not compatible with flake8 3.8
Attempting to run flake8-pyi with flake8 3.8 (which hasn't been released yet) raises an AttributeError due to the use of `optparse` instead of `argparse`.

```
$ pip install flake8==3.8.0a2 flake8-pyi
$ flake8
...
File ".../lib/python3.8/site-packages/pyi.py", line 359, in add_options
    option.to_optparse().default = option.default
File ".../lib/python3.8/site-packages/flake8/options/manager.py", line 332, in to_optparse
    raise AttributeError("to_optparse: flake8 now uses argparse")
AttributeError: to_optparse: flake8 now uses argparse
```

### PR DESCRIPTION
**Pull request by bryanforbes**
Fixes #35

### CODE REVIEW

File: pyi.py [hunk 1]
```diff
@@ -365,12 +366,15 @@ def run(self):
...
     def add_options(cls, parser):
         """This is brittle, there's multiple levels of caching of defaults."""
-        for option in parser.options:
-            if option.long_option_name == "--filename":
-                option.default = "*.py,*.pyi"
-                option.option_kwargs["default"] = option.default
-                option.to_optparse().default = option.default
-                parser.parser.defaults[option.dest] = option.default
+        if isinstance(parser.parser, argparse.ArgumentParser):
+            parser.parser.set_defaults(filename="*.py,*.pyi")
```
Comment (JelleZijlstra) [inline pyi.py:370]: wow this new code is so much nicer

**Comment by JelleZijlstra** [comment]
Thank you!

**Comment by bryanforbes** [comment]
Is there a chance we could get this in a release soon?

**Comment by JelleZijlstra** [comment]
Let me see if I have that power. Somebody else reported this so we better fix it soon :)

**Comment by JelleZijlstra** [comment]
https://pypi.org/project/flake8-pyi/#history 20.5.0 is out!

**Pull request merged**